src/store/current-game/main-playable-actions.store.ts

```diff
@@ -78,18 +81,18 @@ export const MainPlayableActionsStoreModule: Module<
 
       ruleEffects.forEach((ruleEffect) => {
         const gameTurnNumber = rootState.currentGame!.turnNumber;
-        switch (ruleEffect.type) {
-          case RuleEffectType.ADD_GRELOTTINE:
+        switch (ruleEffect.event) {
+          case RuleEffectEvent.ADD_GRELOTTINE:
             commit("currentGame/addGrelottine", ruleEffect.playerName, {
               root: true,
             });
             return;
```

Comment:
With the new events, we may want to remove those `returns` in order to add to the player's history when a grelottine is added/removed.